src/loaders/workspace_downloader/workspace_downloader_helper.py

```diff
@@ -0,0 +1,94 @@
+import os
+import time
+import uuid
+from multiprocessing import Pool, Queue
+from typing import Callable
+
+import docker
+
+from src.clients.AssemblyUtilClient import AssemblyUtil
+from src.clients.SampleServiceClient import SampleService
+from src.clients.workspaceClient import Workspace
+from src.loaders.common import loader_helper
+
+class Conf:
+    def __init__(
+            self,
+            job_dir: str,
+            output_dir: str,
+            kb_base_url: str,
+            token_filepath: str,
+            image_str: str,
+            workers: int,
```

Comment:
```suggestion
            kb_base_url: str = <ci url>,
            token_filepath: str = None,
            image_str: str = <default image string>,
            workers: int = 5,
```

Reply:
Updated `CALLBACK_IMAGE_NAME` as **kbase/callback:test**. 
Tested and it worked for downloader as well.

Reply:
:+1: